src/engine/dataapi/DataStore.java

```diff
@@ -103,6 +103,37 @@ public UserDataStream readInputData(String filePathIn, char delim, String filePa
         
         return data;
 	}
+	
+	/**
+	 * This method is used for the end to end test in order to keep everything else working.
```

Comment:
If we had more time, this is definitely a good impl and might sub the old one and rework apis.